common/rendermode.h

```diff
@@ -54,8 +54,8 @@ enum RenderMode {
 	kRenderHercA = 5,
 	kRenderAmiga = 6,
 	kRenderFMTowns = 7,
-	kRenderPC9821 = 8,
-	kRenderPC9801 = 9,
+	kRenderPC98_256c = 8,
```

Comment:
Unit tests need to be updated with this; this is currently breaking the github builds. See `test/common/rendermode.h`

Reply:
Okay, I have too look into this. I have absolutely no clue what that is about. Probably something new...

Reply:
It isn't actually new. It can be seen from the things that have been included. All render modes that came later than kRenderPC9821 and kRenderPC9801 aren't in...